app/settings.py

```diff
@@ -15,3 +15,9 @@
 RABBIT_PORT = '5672'
 HEALTHCHECK_DELAY_MS = '5000'
 HEARTBEAT_INTERVAL = "?heartbeat_interval=5"
+
+# AWS CORP Settings
+AWS_CORP_RABBITMQ_PORT = '5672',
+AWS_CORP_RABBIT_HOST = os.getenv('AWS_RABBITMQ_HOST', '127.0.0.1')
+AWS_CORP_USER = os.getenv('AWS_CORP_RABBITMQ_DEFAULT_USER', 'guest')
+AWS_CORP_PASSWORD = ('AWS_CORP_RABBITMQ_DEFAULT_PASS', 'guest')
```

Comment:
I think the defaults for username and password should match too, i.e. rabbit / rabbit

Reply:
Also AWS_CORP_PASSWORD is missing os.getenv

should be:

AWS_CORP_PASSWORD = os.getenv('AWS_CORP_RABBITMQ_DEFAULT_PASS', 'guest')